docs/Changelog.md

```diff
@@ -13967,12 +13967,18 @@ This version of the operator has been available since version 12 of the default
 
 ### <a name="BatchNormalization-12"></a>**BatchNormalization-12**</a>
 
-  Carries out batch normalization as described in the paper
-  https://arxiv.org/abs/1502.03167. Depending on the mode it is being run,
-  there are multiple cases for the number of outputs, which we list below:
+  Carries out batch normalization as described in the paper https://arxiv.org/abs/1502.03167.
+  There is three required inputs 'X', 'mean' and 'var', in addition to one optional input 'training_mode'.
+  Note that 'mean' and 'var' are expected to be the estimated statistics in inference mode (training_mode=False, default),
+  and the running statistics in training mode (traning_mode=True).
+  There is one required output 'Y' and four optional outputs : 'mean', 'var', 'saved_mean', 'saved_var' used for training.
   
-  Output case #1: Y, mean, var, saved_mean, saved_var (training mode)
-  Output case #2: Y (test mode, where other outputs will not be populated)
+  The statistics are updated as follows:
+  ```
+  mean = running_mean * momentum + saved_mean * (1 - momentum)
+  var = running_var * momentum + saved_var * (1 - momentum)
+  ```
+  where 'saved_mean' and 'saved_var' are the observed mean and var per channel of the input X.
```

Comment:
From the description here, it looks like `saved_mean = ReduceMean(X, axis=channel_index)`. Is it correct?

Reply:
actually saved_mean=reducemean(X, axis=all_except_channel_index)

for example : if the input size is (1, 2, 1, 3), the nb channel is at index 1, and nb_channel = 2. 
So saved_mean is reduced over indices (0,2,3), and saved_mean should be of size (2).

Reply:
Thanks a lot for the nice explanation. May we add it to the spec? As you have seen, I am a poor user who misinterpret this sentence. For a mathematical operator, I feel it's always better to write done things in a mathematical way. :)

Reply:
thanks! I updated the doc